README.md

```diff
@@ -41,6 +41,18 @@ dependencies {
 }
 ```
 
+and add the Fabric repository
```

Comment:
@marceloslacerda I agree that this needs to be in the main README because it's part of the installation process, but I think it would be better to link to the sub README like so:
```
If you're including the `firebase-ui-auth` dependency, there's a little [more setup](https://github.com/firebase/FirebaseUI-Android/tree/master/auth#configuration) required.
```